tests/debug_api.rs

```diff
@@ -201,6 +203,22 @@ async fn test_raw_transactions(#[future] katana: Katana, _setup: ()) {
     let (server_addr, server_handle) =
         start_kakarot_rpc_server(&katana).await.expect("Error setting up Kakarot RPC server");
 
+    // Get the first transaction from the mock data.
+    let tx = &katana
+        .mock_data
+        .get(&CollectionDB::Transactions)
+        .unwrap()
+        .first()
+        .unwrap()
+        .extract_stored_transaction()
+        .unwrap()
+        .tx;
```

Comment:
can't this be `get_transaction`? I think you implemented `get_transaction` but never used it

Reply:
That's true, just fixed.